mawabot/client.py

```diff
@@ -78,9 +72,24 @@ def run(self):
         else:
             self.output_chan = self.get_channel(int(self.config['output-channel']))
 
-        for cog in COGS:
-            self.load_extension(cog)
-            logger.info(f'Loaded cog: {cog}')
+        self.add_cog(Reloader(self))
+        logger.info('Loaded cog: Reloader')
+
+        files = [file.replace('.py', '') for file in listdir(f'mawabot/cogs')
```

Comment:
I feel like a regex should be used here. While unlikely to happen, this would mess up a cog that starts with "py".

```python
PY_FILE_REGEX = re.compile(r'\.py', re.IGNORECASE)
# ...
files = [PY_FILE_REGEX.sub('', file) for file in listdir('mawabot/cogs') if '.py' in file]
```